iscript/src/iscript/macvpn.py

```diff
@@ -305,7 +296,40 @@ async def vpn_behavior(config, task, notarize=True):
         provisionprofile_filename="orgmozillamacosFirefoxVPN.provisionprofile",
     )
 
+    return [top_app]
+
+
+async def vpn_behavior(config, task, notarize=True):
+    """Notarize vpn app.
+
+    Workflow:
+    . Sign all inner apps
+    . Sign main app
+    . Create pkg
+    . Notarize and staple pkg (optional)
```

Comment:
We're notarizing in a different task. As it stands right now, iscript notarizations are deprecated.